### PR TITLE
Remove optimized table setting

### PR DESCRIPTION
**Pull request by Charles-Gagnon**
Removing this for now as it causes issues such as in https://github.com/microsoft/azuredatastudio/issues/16844

### CODE REVIEW

**Review by alanrenmsft — APPROVED** [review]

**Review by kburtram — APPROVED** [review]
Should we also remove the unused alternate table implementation?

**Comment by Charles-Gagnon** [comment]
I figure we'd keep it in for ease of continuing work on this in the future. But if that doesn't seem like something we're likely to do then I have no issue removing it completely. @kburtram I'll let that be your call

**Comment by Charles-Gagnon** [comment]
After thinking a bit more, I'm just going to remove it. Not sure how much context we'd be able to get out of it anyways and it's likely just to drift further from the codebase as time goes on.

@lucyzhang929 If you're still looking at perf stuff maybe take a look at what Anthony was doing here for ideas

File: src/sql/workbench/contrib/query/browser/query.contribution.ts [hunk 1]
```diff
@@ -392,11 +392,13 @@ const queryEditorConfiguration: IConfigurationNode = {
 			'description': localize('queryEditor.results.copyRemoveNewLine', "Configuration options for copying multi-line results from the Results View"),
 			'default': true
 		},
+		/* Removing this for now until work can be completed on it
```
Comment (alanrenmsft) [inline src/sql/workbench/contrib/query/browser/query.contribution.ts:395]: this can be removed too.

**Pull request merged**